package.json

```
@@ -99,7 +99,7 @@
     "start:services": "docker-compose up",
     "build": "yarn typecheck && vite build",
     "prettier": "prettier --write \"src/**/*.{js,jsx,ts,tsx}\"",
-    "lint": "eslint src/ cypress/",
+    "lint": "eslint \"./src\" \"./cypress\" --ext .js,.jsx,.ts,.tsx",
```

Nice catch. Thanks for fixing this! I've tried this locally and it should work as follows:

```suggestion
    "lint": "eslint src/ cypress/ --ext .js,.jsx,.ts,.tsx",
```

It should work on mac as well, but let me know if it doesn't. We could even remove the trailing slashes, but I think it's easier to understand these are paths this way.

---

Indeed, fixed!